www/include/common/javascript/datepicker/localizedDatepicker.js

```diff
@@ -134,26 +135,29 @@ function turnOffEvents() {
     $(".timepicker").off('change');
 }
 
+
 function updateEndTime() {
+    var durationScale = { "d" : 86400, "h" : 3600, "m": 60, "s": 1 };
```

Comment:
The duration scale should be set by the back end, not the front

Reply:
I think this is more or less related to to the first conversation above (https://github.com/centreon/centreon/pull/7606#discussion_r351911009), as here I use  _Duration_ scale value from _Administration > Parameters > Monitoring_...

Reply:
Done, duration scale settings removed from the front.